diff --git a/src/vs/workbench/services/extensionManagement/browser/extensionEnablementService.ts b/src/vs/workbench/services/extensionManagement/browser/extensionEnablementService.ts
index 10b674d..d6b8ab4 100644
--- a/src/vs/workbench/services/extensionManagement/browser/extensionEnablementService.ts
+++ b/src/vs/workbench/services/extensionManagement/browser/extensionEnablementService.ts
@@ -276,19 +276,11 @@ export class ExtensionEnablementService extends Disposable implements IWorkbench
 
 				const index = extensionsToEnable.findIndex(e => areSameExtensions(e.identifier, extension.identifier));
 
-				// Extension is not added to the disablement list so add it
-				if (index === -1) {
-					extensionsToEnable.push(extension);
-				}
-
-				// Extension is there already in the disablement list.
-				else {
-					try {
-						// Replace only if the enablement state can be changed
-						this.throwErrorIfEnablementStateCannotBeChanged(extension, enablementStateOfExtension, true);
-						extensionsToEnable.splice(index, 1, extension);
-					} catch (error) { /*Do not add*/ }
-				}
+				try {
+					// Replace only if the enablement state can be changed
+					this.throwErrorIfEnablementStateCannotBeChanged(extension, enablementStateOfExtension, true);
+					extensionsToEnable.splice(index, 1, extension);
+				} catch (error) { /*Do not add*/ }
 			}
 		}
 
